reference/manifest/action.md

```diff
@@ -64,6 +64,33 @@ The following example shows two Actions that share the same **TaskpaneId**.
 </Action>
 ```  
 
+## Title
+Optional element when  **xsi:type** is "ShowTaskpane". Specifies the custom title for the task pane for this action. 
+
+The following examples show two different Actions that use the **Title** element. To see the examples in their entirety, see [Script Lab manifest](https://github.com/OfficeDev/script-lab/blob/master/manifests/script-lab-local.xml)
```

Comment:
Please lowercase Action in this sentence, as you are referring to the action, not the element.